utils/check_modules.sh

```diff
@@ -36,40 +36,37 @@
 # (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
 # THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 
-set -e
```

Comment:
I think the exit code of this script is ignored anyway.  check_script.py returns an error if there are any warnings, but we could easily remove that and set -e here which would cause exit-on-error to work, but that doesn't appear to have any effect.